### PR TITLE
Added Basic Audit Rules

### PR DESCRIPTION
**Pull request by ankit-privado**
Added Basic Audit Collections, Webclient, and Utility Rules.

### CODE REVIEW

File: config/audit/utility/java.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  - id: AuditUtility.CommonsIo
+    name: Commons Io
+    patterns:
+      - '^commons-io.*:commons-io.*'
+
+  - id: AuditUtility.ApacheLogging
+    name: Apache Logging Log4j
+    patterns:
+      - '^org.apache.logging.log4j.*:log4j-core.*'
```
Comment (pandurangpatil) [inline config/audit/utility/java.yaml:20]: These are leakage libraries. These will not come under utility libraries

File: config/audit/utility/java.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  - id: AuditUtility.Slf4j
+    name: Slf4j
+    patterns:
+      - '^org.slf4j.*:slf4j-api.*'
```
Comment (pandurangpatil) [inline config/audit/utility/java.yaml:25]: This one also leakage library

**Review by pandurangpatil — CHANGES_REQUESTED** [review]
Collections and webclient look good. For utility, we need to add more libraries.

File: config/audit/utility/java.yaml (hunk 1 above)
Comment (ankit-privado) [reply config/audit/utility/java.yaml:20]: okay removing it.

File: config/audit/utility/java.yaml (hunk 2 above)
Comment (ankit-privado) [reply config/audit/utility/java.yaml:25]: okay

**Review by pandurangpatil — APPROVED** [review]
looks good

**Review by pandurangpatil — APPROVED** [review]
looks good

**Pull request merged**